README.md

```diff
@@ -61,13 +64,15 @@ _Note: may need to make **auto** folder_
 
 ##### Install build tools
 
-Debian/Ubuntu/Mint:   `$ sudo apt-get install build-essential libXmu-dev libXi-dev libgl-dev cmake`
+Debian/Ubuntu/Mint:   `$ sudo apt-get install build-essential libxmu-dev libxi-dev libgl-dev cmake`
 
 RedHat/CentOS/Fedora: `$ sudo yum install libXmu-devel libXi-devel libGL-devel cmake`
 
 ##### Build
 
-	$ cd build
+	$ cd auto
```

Comment:
It's intended to build GLEW from pre-built sources in the .tgz or .zip release, rather than "from scratch" from github master branch.

Reply:
I'm sorry. I misunderstood the instructions. I though that these was for building from scratch.